.travis.yml

```diff
@@ -1,5 +1,44 @@
 sudo: required
 dist: trusty
+
+env:
+  COMPOSE_VERSION: 1.24.1
+
+before_install:
+  # Docker Compose Install
+  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  # Download and install Docker libs
+  - curl -L https://github.com/Ortus-Solutions/docker-buildfiles/archive/master.zip > docker.zip
+  - unzip docker.zip -d workbench
+  - mv workbench/docker-buildfiles-master workbench/docker
+  # CommandBox Keys
+  - sudo apt-key adv --keyserver keys.gnupg.net --recv 6DA70622
+  - sudo echo "deb http://downloads.ortussolutions.com/debs/noarch /" | sudo tee -a
+    /etc/apt/sources.list.d/commandbox.list
+  - git clone https://github.com/DSpace-Labs/DSpace-Docker-Images.git
+
+install:
+  # Core testing install
+  - sudo apt-get update && sudo apt-get --assume-yes install commandbox
+  - box install
+  - box server start
+  # Docker CFML Server Tests
+  - export ANGULAR_SRC=$(pwd)
+  - cd DSpace-Docker-Images/docker-compose-files/dspace-compose
+  - docker-compose -f d7.travis.yml up -d
```

Comment:
Per @artlowel's notes about Travis not being able to find the final coveralls files...I wonder if part of the problem could be the `cd` command just before this line?  It's possible Travis is running most commands from the `dspace-compose` subfolder (since you never move back up to the top directory), which would cause some odd behaviors.

I wonder if we should collapse these two lines into one?  For example, it seems like you could just run
`docker-compose -f DSpace-Docker-Images/docker-compose-files/dspace-compose/d7.travis.yml up -d`

You'd also want to modify the `after_script` though too, so that its `docker-compose` uses the full path.  That might mean we'd want to create an environment variable here...e.g. `export DSPACE_COMPOSE_DIR=$(pwd)/DSpace-Docker-Images/docker-compose-files/dspace-compose` and then use that variable where needed.